contact/style.scss

```
@@ -0,0 +1,143 @@
+body{
+  background-color: #FFFFFF;
+  margin: 36px;
+}
+
+h1{
+  height: 22px;
+  text-align: right;
+  hirgt:54px;
+}
+
+.content-page-title{
+  color: #000000;
+  text-align: center;
+  hirgt:77px;
+}
+
+
+
+
+
+.contact-primary{
+  p{
+    color: #707070;  
+    text-align: center;
+    height: 107px;
+    margin: 36px;
+    }
+
+  ul{
+  display: block;
```

```suggestion
  display: block;
```
ul は、display: block; 無しで大丈夫です。